Adyen/UI/Form/Items/Address/AddressViewModel.swift

```diff
@@ -17,72 +18,30 @@ internal enum AddressField: String, CaseIterable {
     case country
 }
 
-internal enum FormScheme {
+/// :nodoc:
+public enum FormScheme {
```

Comment:
Maybe `AddressFormScheme` ?

Reply:
I have renamed this. thanks for pointing it out.